docs/diagnostics/PX1020.md

```diff
@@ -0,0 +1,31 @@
+# PX1020
+This document describes the PX1020 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                                                | Type  | Code Fix    | 
+| ------ | ---------------------------------------------------------------------------------------------------------------- | ----- | ----------- | 
+| PX1020 | For the DAC property with the `AutoNumber` attribute, the string length must be greater than or equal to 15.     | Error | Available   |
+
+## Diagnostic Description
+You can use the `AutoNumber` attribute on a DAC field to enable auto-numbering for the field. The string must be of the string type and the field length you specify in the `PXDBString` attribute for the field must be at least 15 symbols.
+
```

Comment:
Here we need more details. First of all, the sufficient length is not a hard-coded number, 15. It is actually taken from `PX.Objects.CS.NumberingSequence` DAC's field `StartNbr`. For autonumbering to work correctly the length of the auto-numbered field can't be lesser than the length of a start  number of numbering sequence which is described by the `StartNbr` field. Currently, its length is 15, but its not hard-coded, the check actually obtains the value from the `NumberingSequence` DAC. 
We need to include this into the docs to explain users why the alert is displayed.